README.ja.md

```
@@ -0,0 +1,74 @@
+LikeC4 は、ソフトウェアアーキテクチャを記述するためのモデリング言語であり、モデルから図を生成するためのツール群です。
+
+LikeC4 は [C4 Model](https://c4model.com/) と [Structurizr DSL](https://github.com/structurizr/dsl) に着想を得ながら、一定の柔軟性も備えています。
+独自の表記法や要素タイプ、アーキテクチャモデル内の任意の数の入れ子階層をカスタマイズまたは定義できます。\
+ニーズにぴったり合わせられます。
+
+## LikeC4 はどのようなもの？
+
+LikeC4 のソース：
+
+<div align="center">
+  <img src="https://github.com/likec4/.github/assets/824903/c0f22106-dba6-469e-ab47-85e7b8565513" width="675px">
```

markdownlint-cli2 (0.23.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)